patches/chromium/fix_crash_on_nativetheme_change_during_context_menu_close.patch

```diff
@@ -1,42 +0,0 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Shelley Vohr <shelley.vohr@gmail.com>
-Date: Fri, 16 Jun 2023 11:15:57 +0200
-Subject: fix: crash on nativeTheme change during context menu close
-
-Refs https://chromium-review.googlesource.com/c/chromium/src/+/4363385
-
-Fixes a crash seen when trying to change nativeTheme source during
-a context menu close. This happens as a result of a change added in the
-above CL, which doesn't check whether or not the menu controller could
-possibly be null as can happen during a context menu close. This
-only affects Windows 11, as Bubble Border is only enabled there.
-
-This should be upstreamed, as other uses of MenuController in this
-file do check for menu controller being null.
-
-diff --git a/ui/views/controls/menu/menu_scroll_view_container.cc b/ui/views/controls/menu/menu_scroll_view_container.cc
-index bbed53e48bea74f6fe44225e23ac27af34e4cabd..103c303979928cd8f40e543c3cafbd2cc0374d22 100644
---- a/ui/views/controls/menu/menu_scroll_view_container.cc
-+++ b/ui/views/controls/menu/menu_scroll_view_container.cc
-@@ -391,8 +391,7 @@ void MenuScrollViewContainer::CreateDefaultBorder() {
-   MenuController* menu_controller =
-       content_view_->GetMenuItem()->GetMenuController();
-   const MenuConfig& menu_config = MenuConfig::instance();
--  corner_radius_ = menu_config.CornerRadiusForMenu(
--      content_view_->GetMenuItem()->GetMenuController());
-+  corner_radius_ = menu_config.CornerRadiusForMenu(menu_controller);
-   int padding = menu_config.use_outer_border && corner_radius_ > 0
-                     ? kBorderPaddingDueToRoundedCorners
-                     : 0;
```

Comment:
Looks like this was removed due to https://chromium-review.googlesource.com/c/chromium/src/+/4762986